grub-core/tpm2/module.c

```diff
@@ -757,12 +757,33 @@ static grub_err_t
 grub_tpm2_protector_nv_recover (const struct grub_tpm2_protector_context *ctx,
 				grub_uint8_t **key, grub_size_t *key_size)
 {
-  (void)ctx;
-  (void)key;
-  (void)key_size;
+  TPM_HANDLE sealed_handle = ctx->nv;
+  tpm2key_policy_t policy_seq = NULL;
+  grub_err_t err;
+
+  /*
+   * Create a basic policy sequence based on the given PCR selection if the
+   * key file doesn't provide one
+   */
+  if (policy_seq == NULL)
```

Comment:
We probably can ignore this check since `policy_seq` is always `NULL` in this function.

Reply:
Ah yes, good catch.